doc/api/net.md

```diff
@@ -416,109 +571,134 @@ See also: [`socket.setTimeout()`][]
+* `path`: Path the client should connect to
+* `connectListener` {Function}
 
-Normally this method is not needed, as `net.createConnection` opens the
-socket. Use this only if you are implementing a custom Socket.
+The `socket.connect()` method opens the connection for a given socket using
+the given `path`.
 
-This function is asynchronous. When the [`'connect'`][] event is emitted the
-socket is established. If there is a problem connecting, the `'connect'` event
-will not be emitted, the [`'error'`][] event will be emitted with the exception.
+This method operates asynchronously. The [`'connect'`][] event is emitted when
+the connection is successfully established. If an error occurs while attempting
+to open the connection, the [`'error'`][] event will be emitted.
 
-The `connectListener` parameter will be added as a listener for the
-[`'connect'`][] event.
+If the `connectListener` argument is given, the callback will be added as a
+listener for the [`'connect'`][] event.
 
-### socket.connect(path[, connectListener])
 ### socket.connect(port[, host][, connectListener])
 <!-- YAML
 added: v0.1.90
 -->
 
-As [`socket.connect(options[, connectListener])`][`socket.connect(options, connectListener)`],
-with options either as either `{port: port, host: host}` or `{path: path}`.
+* `port` {number} Port the client should connect to
+* `host` {String} Host the client should connect. to Defaults to `'localhost'`
+* `connectListener` {Function}
+
+The `socket.connect()` method opens the connection for a given socket using
+the given `port` and `host`.
+
+This method operates asynchronously. The [`'connect'`][] event is emitted when
+the connection is successfully established. If an error occurs while attempting
+to open the connection, the [`'error'`][] event will be emitted.
+
+If the `connectListener` argument is given, the callback will be added as a
+listener for the [`'connect'`][] event.
 
 ### socket.connecting
 <!-- YAML
 added: v6.1.0
 -->
 
-If `true` - [`socket.connect(options[, connectListener])`][`socket.connect(options, connectListener)`] was called and
-haven't yet finished. Will be set to `false` before emitting `connect` event
-and/or calling [`socket.connect(options[, connectListener])`][`socket.connect(options, connectListener)`]'s callback.
+A boolean property that when `true`, indicates that `socket.connect()` has been
+called a connection is in the process of being established. The value will be
+set to `false` once the connection is established but *before* the `'connect'`
+event is emitted. The value will also be `false` before `socket.connect()` is
+called.
 
 ### socket.destroy([exception])
 <!-- YAML
 added: v0.1.90
 -->
 
-Ensures that no more I/O activity happens on this socket. Only necessary in
-case of errors (parse error or so).
+The `socket.destroy()` method ensures that no futher I/O activity can occur
+with the socket. This is typically only necessary when errors occur.
```

Comment:
this has always been under described. Will it do a FIN or a RST to the remote peer? or neither?